src/main/java/teammates/ui/controller/StudentCommentsPageData.java

```diff
@@ -51,4 +113,144 @@ public String getRecipientNames(Set<String> recipients){
         String namesString = namesStringBuilder.toString();
         return removeEndComma(namesString);
     }
+    
+    
+    private void setCoursePagination(List<String> coursePaginationList) {
+        String previousPageLink = retrievePreviousPageLink(coursePaginationList);
+        String nextPageLink = retrieveNextPageLink(coursePaginationList);
+        String activeCourse = "";
+        for (String courseId : coursePaginationList) {
+            if (courseId.equals(this.courseId)) {
+                activeCourse = courseId;
+            }
+        }
```

Comment:
`String activeCourse = courseId`? Or just use `courseId` directly?
Or if you want to ensure that `courseId` is inside the pagination list:
```
if (coursePaginationList.contains(courseId)) {
    activeCourse = courseId;
}
```